### ISSUE TITLE
Error `this.quillJS.getLine is not a function`

### ISSUE THREAD

**Issue by vadimkantorov** [cloverhearts/quilljs-markdown#74]
Hi @cloverhearts !

I'm setting up Quill + QuillMarkdown as follows:
```html
<html>
    <body>
        <head>
            <link href="https://cdn.quilljs.com/1.0.0/quill.snow.css" rel="stylesheet" />
            <script src="https://cdn.quilljs.com/1.0.0/quill.js"></script>
            <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/quilljs-markdown@latest/dist/quilljs-markdown-common-style.css" />
            <script src="https://cdn.jsdelivr.net/npm/quilljs-markdown@latest/dist/quilljs-markdown.js"></script>
        </head>

        <div id="editor"></div>
        <script>
            const editor = new Quill('#editor', { theme: 'snow' });
            const editor_markdown = new QuillMarkdown(editor);
        </script>
    </body>
</html>
```

This loads without errors in Chrome DevConsole.

However, when I start typing `# hello`, I get this strange result: 
- no header formatting
- more identical errors at every char typed
- the reversed output: ` hello#` instead of `# hello`

<img width="958" alt="image" src="https://github.com/cloverhearts/quilljs-markdown/assets/1041752/dd9f9660-375b-4a47-b327-a4e0a242c5a8">

```
app.js:34 Uncaught TypeError: this.quillJS.getLine is not a function
    at t.value (app.js:34:39)
    at Emitter.emit (quill.js:4316:28)
    at Emitter.emit (quill.js:4146:100)
    at Editor.update (quill.js:4046:44)
    at Emitter.emit (quill.js:4310:34)
    at Emitter.emit (quill.js:4146:100)
    at Scroll.update (quill.js:6299:23)
    at MutationObserver.<anonymous> (quill.js:1401:20)
value @ app.js:34
emit @ quill.js:4316
emit @ quill.js:4146
update @ quill.js:4046
emit @ quill.js:4310
emit @ quill.js:4146
update @ quill.js:6299
(anonymous) @ quill.js:1401
```

How could I get this basic example to work?

Thank you very much :)

---

Basically, my usecase is creating a simple WYSIWYG markdown editor for files stored at GitHub. Is this secnario possible? Does quilljs-markdown preserve the underlying Markdown? Or is it not possible to retrieve it? I saw https://github.com/cloverhearts/quilljs-markdown/issues/70#issuecomment-1528716904, but don't understand completely the limitation you described

**Comment by vadimkantorov** [comment]
Okay, it appears it's because I followed Quill's README a bit too well and used Quill version `1.0.0` instead of `1.3.7`: https://github.com/quilljs/quill/issues/3932

But maybe it would be great if quilljs-markdown README included this complete HTML example

**Comment by cloverhearts** [comment]
Thank you for your report.
I'll fix it soon. 👍
